src/main/java/OderAgain.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Scanner;
+
+public class OderAgain {
+
+    Scanner sc = new Scanner(System.in);
+
+    int OderAgainQuestion() {
```

Comment:
🍏 Отсюда можно возвращать булевую переменную true или false

```suggestion
    boolean orderAgain() {
```